### DISCUSSION TITLE
Credits Based System

### DISCUSSION THREAD

**Discussion by Psyf** [Glench/ExtPay#82] [Ideas]
Hi! 

Tldr; Any plans/workaround for points-based extension?

**Context:** 
I stumbled upon ExtPay when I was digging around for an easy way to add auth to my browser extension. 

My original plan: 
- Use a low-code platform to enabled auth on the extension and get a uuid when user logs in. 
- Ask user to pay for points/credits. I'd essentially integrate stripe and pass in the uuid for tracking. 
- Stripe successful payment callback would hit my server, where uuid would get points incremented. 
- When user performs an action via the extension, they pay with points.

However, I couldn't find a low-code platform to integrate into my extension easily. (Auth0 doesn't have official support, and it's a bad DX and PITA for cross-browser support). 

If ExtPay had something like `user.credits` (which payment would increment, and server/APIKey would decrement on usage), my plan would become: 
- integrate ExtPay. 

Thoughts?
